library/network_connections.py

```diff
@@ -1589,7 +1600,7 @@ def fail_json(
         kwargs["msg"] = msg
         kwargs["changed"] = changed
         self.module.fail_json(
-            **self._complete_kwargs(connections, kwargs, traceback_msg)
+            **self._complete_kwargs(connections, kwargs, traceback_msg, True)
```

Comment:
To make it more clear what `True` refers to:

```suggestion
            **self._complete_kwargs(connections, kwargs, traceback_msg, fail=True)
```

Reply:
I've fixed it now